### PR TITLE
Replace PHP_CodeSniffer with PHP-CS-fixer.

### LINKED ISSUE #107: OpenSky Symfony2-coding-standard repo is 404
From CONTRIBUTING.md

> This library will use the Symfony2 Coding Standard. The easiest way to apply these conventions is to install PHP_CodeSniffer and the [Opensky Symfony2 Coding Standard](http://symfony.com/doc/current/contributing/code/standards.html).

The [OpenSky Symfony2 coding standard repository] is 404. Is there another set of PHP_CodeSniffer rules that might replace the OpenSky rules?

### PR DESCRIPTION
**Pull request by jeremykendall**
Fixes #107.

@toin0u You should "tune" the `.php_cs` config file to specify any fixers you'd like added or removed ([PHP-CS-Fixer usage docs](https://github.com/FriendsOfPHP/PHP-CS-Fixer#usage)). Here's a sample that I use on one of the Graph Story projects:

``` php
<?php

require_once './vendor/autoload.php';

$finder = \Symfony\CS\Finder\DefaultFinder::create()
    ->in('src/');

return \Symfony\CS\Config\Config::create()
    ->setUsingCache(true)
    ->fixers([
        '-single_blank_line_before_namespace',
        '-pre_increment',
        '-concat_without_spaces',
        'concat_with_spaces',
        'ordered_use',
    ])
    ->finder($finder);
```

Without tuning the fixers, this is what gets changed: 

``` bash
$ php-cs-fixer fix --verbose
Loaded config from "/Users/jeremykendall/dev/DigitalOceanV2/.php_cs"
..F.FF....FF......F....F.....F..F..F..
Legend: ?-unknown, I-invalid file syntax, file ignored, .-no changes, F-fixed, E-error
   1) Adapter/BuzzAdapter.php (unalign_double_arrow)
   2) Adapter/Guzzle5Adapter.php (unalign_double_arrow, unalign_equals)
   3) Adapter/GuzzleAdapter.php (unalign_double_arrow, unalign_equals)
   4) Api/DomainRecord.php (multiline_array_trailing_comma)
   5) Api/Droplet.php (multiline_array_trailing_comma, single_quote, phpdoc_params)
   6) Entity/AbstractEntity.php (braces)
   7) Entity/Droplet.php (single_quote)
   8) Entity/NextBackupWindow.php (phpdoc_short_description)
   9) Entity/Size.php (phpdoc_scalar)
  10) Exception/ExceptionReader.php (unalign_equals)
Fixed all files in 5.424 seconds, 7.750 MB memory used
```

### CODE REVIEW

File: .php_cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+require_once './vendor/autoload.php';
+
+$finder = \Symfony\CS\Finder\DefaultFinder::create()
```
Comment (GrahamCampbell) [inline .php_cs]: no leading slash please

File: .php_cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    ->in('src/');
+
+return \Symfony\CS\Config\Config::create()
```
Comment (GrahamCampbell) [inline .php_cs]: same

**Comment by toin0u** [comment]
@jeremykendall Thanks! :+1: Can you remove leading slahes?

**Comment by jeremykendall** [comment]
@toin0u && @GrahamCampbell Done!

File: composer.json [hunk 3]
```diff
@@ -34,7 +34,8 @@
         "kriswallsmith/buzz": "~0.10",
         "guzzle/guzzle"     : "~3.7",
         "guzzlehttp/guzzle" : "~5.0",
-        "phpspec/phpspec"   : "~2.0"
+        "phpspec/phpspec"   : "~2.0",
+        "fabpot/php-cs-fixer": "^1.10"
```
Comment (GrahamCampbell) [inline composer.json]: please use `~`

File: .php_cs [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+require_once './vendor/autoload.php';
+
+use Symfony\CS\Finder\DefaultFinder;
+use Symfony\CS\Config\Config;
+
+$finder = DefaultFinder::create()
+    ->in('src/');
+
+return Config::create()
```
Comment (GrahamCampbell) [inline .php_cs]: could you explicitly set the level to symfony here so the move to 2.x will be smooth

**Comment by jeremykendall** [comment]
Ping

**Comment by GrahamCampbell** [comment]
:heart:

**Comment by toin0u** [comment]
@jeremykendall Thank you! <3

**Comment by jeremykendall** [comment]
Thanks, guys!

**Pull request merged**